ipaserver/dnssec/ldapkeydb.py

```diff
@@ -467,7 +467,7 @@ def zone_keypairs(self):
 
     # LDAP initialization
     dns_dn = DN(ipalib.api.env.container_dns, ipalib.api.env.basedn)
-    ldap = ipaldap.LDAPClient(ipalib.api.env.ldap_uri)
+    ldap = ipaldap.LDAPClient(ipalib.api.env.ldap_uri, start_tls=True)
     logger.debug('Connecting to LDAP')
```

Comment:
A similar point. This one explicitly uses ldap_uri which IPA sets as ldapi. Is this to prevent users who might muck around with things from making an insecure connection? Because otherwise one might consider start_tls to conflict with ldapi (at best it is a no-op).

Reply:
I removed the ``start_tls=True`` argument again. The LDAP key DB for DNSSEC uses GSSAPI and is therefore not vulnerable.